src/payment/cart/Cart.jsx

```diff
@@ -78,9 +89,11 @@ class Cart extends React.Component {
             <CouponForm isPaymentVisualExperiment={isPaymentVisualExperiment} /> : null}
           <TotalTable total={orderTotal} />
           {isCurrencyConverted ? <CurrencyDisclaimer /> : null}
-        </OrderSummary>
+        </OrderSummary> : (<div><div className="skeleton py-2 mb-3 w-50" />
```

Comment:
You can also add <React.Fragment></React.Fragment> if you don't want to add the extra < div> here, so you can still group multiple children without adding the extra node.